src/Splat.Android.Tests/Splat.Android.Runner.csproj

```diff
@@ -91,12 +95,23 @@
       <Name>Splat</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
+    <AndroidAsset Include="..\Splat.Tests\splatlogo.bmp">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </AndroidAsset>
+    <AndroidAsset Include="..\Splat.Tests\splatlogo.jpg">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Might be worth if we can get the cake script to handle android running if you want to challenge that. We have a repo for the cake recipe which you can run locally the code. I tend to just work with the downloaded version in the tools folder.